blazor/datagrid/cell-edit-types.md

```diff
@@ -984,6 +984,74 @@ The `SfTextBox` component is defined inside the EditTemplate and changes can be
 }
 ```
 
+### Prevent enter key functionality in multiline textbox while editing
+
+By default, while editing a particular row in normal or dialog edit mode, pressing the ENTER key will save the changes made in the particular cell or edit form. Similarly, pressing the ENTER key while performing editing with the Multiline textbox will save the changes. But, in a multiline textbox, instead of this behaviour, it should add a new line break in the text content while pressing the ENTER key. This can be achieved by using the `OnFocus` event of the `SfTextBox`.
+
+In the following sample, Multiline Textbox is rendered in the Customer ID column. The `stopPropagation()` method is called using Microsoft.JSInterop in the `OnFocus` event of the `SfTextBox` to prevent the ENTER key action when editing the Customer ID column.
```

Comment:
In the following sample, the multiline textbox is rendered in the Customer ID column. The `stopPropagation()` method is called using Microsoft.JSInterop in the `OnFocus` event of the `SfTextBox` to prevent the ENTER key action when editing the Customer ID column.